bot/cogs/grades_cog/grades_cog.py

```diff
@@ -0,0 +1,265 @@
+        W = f'{int(df.W.mean().round(2) * 100)}%'
+
+        embeds = []
+
+        embed = discord.Embed(title=f'Grades for {course} since {year}', color=Colors.ClemsonOrange)
+        embed.set_footer(text=self.get_full_name(ctx.author), icon_url=ctx.author.avatar_url)
+        embed.add_field(name='Overall Distribution', value=f'```A: {A}\nB: {B}\nC: {C}\nD: {D}\nF: {F}\nW: {W}```')
+        embed.add_field(name='Total Number of Classes Analyzed', value=len(df), inline=False)
+        embed.add_field(name='Total Number of Professors Found', value=len(df.groupby(['Instructor'])), inline=False)
+        embed.add_field(name='Explanation',value=f'Run `{await self.bot.current_prefix(ctx)}help grades` for more information on this command')
+        embeds.append(embed)
+
+        for i, row in df.groupby(['Instructor']).mean().iterrows():
+            embed = discord.Embed(title=f'Grades for {course} since {year}', color=Colors.ClemsonOrange) 
+            embed.set_footer(text=self.get_full_name(ctx.author), icon_url=ctx.author.avatar_url)
+            A = f'{int(row.A.round(2) * 100)}%'
+            B = f'{int(row.B.round(2) * 100)}%'
+            C = f'{int(row.C.round(2) * 100)}%'
+            D = f'{int(row.D.round(2) * 100)}%'
+            F = f'{int(row.F.round(2) * 100)}%'
+            W = f'{int(row.W.round(2) * 100)}%'
+
+            val = f'```A: {A}%\nB: {B}\nC: {C}\nD: {D}\nF: {F}\nW: {W}```'
+            embed.add_field(name=f"{row.name}'s distribution", value=val)
+            embed.add_field(name='Total Number of Classes Analyzed', value=len(df), inline=False)
+            embed.add_field(name='Total Number of Professors Found', value=len(df.groupby(['Instructor'])), inline=False)
+            embed.add_field(name='Explanation',value=f'Run `{await self.bot.current_prefix(ctx)}help grades` for more information on this command')
+            embeds.append(embed)
+
+        await self.bot.messenger.publish(Events.on_set_pageable_embed, 
+            pages=embeds, 
+            author=ctx.author, 
+            channel=ctx.channel,
+            timeout=360)
+
+    @ext.group(invoke_without_command= True, case_insensitive=True)
+    @ext.long_help(
+        """
+        Attempts to give more information about professor's grade distribution at Clemson.
+
+        DISCLAIMER:
+        Not every professor listed will be at Clemson, this is a tool built for better information but not complete information.
+        In addition, this system works on the Grade Distribution Releases located at https://www.clemson.edu/institutional-effectiveness/oir/data-reports/
+        In Addition, Clemson provides incomplete and mangled professor data so there will be multiple different versions of the same professor. This is just a byproduct of how the data is distributed from the university
+        """
+    )
+    @ext.short_help('Provides info about a given professor')
+    @ext.example(('prof brian dean', 'prof brian dean'))
```

Comment:
Is there a reason to have Brain Dean listed as an example twice in the help documentation?

Reply:
no i messed up the copy pasting